src/pybind/mgr/cephadm/module.py

```diff
@@ -685,7 +685,7 @@ def available(self) -> Tuple[bool, str]:
             return False, 'SSH keys not set. Use `ceph cephadm set-priv-key` and `ceph cephadm set-pub-key` or `ceph cephadm generate-key`'
         return True, ''
 
-    def process(self, completions: List[CephadmCompletion]) -> None:
+    def process(self, completions: List[CephadmCompletion]) -> None:  # type: ignore
```

Comment:
```suggestion
    def process(self, completions: List[orchestrator.Completion]) -> None:  # type: ignore
```

interesting, Liskov substitution principle ... use the super class here and implement the `evaluate` attr?